Installation/Source/openzaak.ztc.source.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Remove this file, already added this in the predesignated file for this: buren.ztc.source.json